docs/migration-guide-1.0.0.md

```diff
@@ -0,0 +1,85 @@
+# Migration Guide `0.x` to `1.0.0`
+
+This migration guide describes how to upgrade existing projects using `next-translate@0.x` to `next-translate@1.0.0`.
+
+1. Update `next-translate` to `^1.0.0`. If you're using a recent version of `npm` you can run this command:
+
+```bash
+npm install next-translate@^1.0.0
```

Comment:
it will be nice to add an example with yarn 😊

Reply:
Okay, I added the yarn example :)